Detour/Source/DetourNavMesh.cpp

```diff
@@ -958,9 +958,9 @@ dtStatus dtNavMesh::addTile(unsigned char* data, int dataSize, int flags,
 		{
 			connectExtLinks(tile, neis[j], -1);
 			connectExtLinks(neis[j], tile, -1);
+			connectExtOffMeshLinks(neis[j], tile, -1);
 		}
 		connectExtOffMeshLinks(tile, neis[j], -1);
-		connectExtOffMeshLinks(neis[j], tile, -1);
```

Comment:
I like that, that makes it a lot clearer that the tile<->tile is a special case. @tfabretti could you make and test that change on your mesh?